src/i18n/messages-fr.po

```diff
@@ -0,0 +1,875 @@
+msgstr "Prendre une photo"
+
+msgid "Upload picture"
+msgstr "Envoyer une photo"
+
+msgid "Waiting for camera"
+msgstr "En attente de la camera"
+
+msgid "Picture"
+msgstr "Avatar"
+
+msgid "The file couldn't be read."
+msgstr "Le fichier ne peux pas être lu"
+
+msgid "The file is not an image."
+msgstr "Le fichier n'est pas une image"
+
+#, python-format
+msgid "The file is too large. Max. %d MB."
+msgstr "Le ficheir est trop gros. Max. %d MB."
+
+msgid "Select file"
+msgstr "Choisir un fichier"
+
+msgid "Chat sessions"
+msgstr "Session de chat"
+
+msgid "Room chat"
+msgstr "Chat de cette salle"
+
+msgid "Peer to peer"
+msgstr "Peer-to-peer"
+
+msgid "Close chat"
+msgstr "FErmer le chat"
+
+msgid "Upload files"
+msgstr "Envoyer les fichiers"
+
+msgid "Share my location"
+msgstr "Partager mon emplacement"
+
+msgid "Clear chat"
+msgstr "Effacer le chat"
+
+msgid "is typing..."
+msgstr " est en train de taper...
+
+msgid "has stopped typing..."
+msgstr " a fini de taper..."
+
+msgid "Type here to chat..."
+msgstr "Taper ici pour chatter..."
+
+msgid "Send"
+msgstr "Envoyer"
+
+msgid "Accept"
+msgstr "Accepter"
+
+msgid "Reject"
+msgstr "Refuser"
+
+msgid "You have no contacts."
+msgstr "Vous n'avez pas de contact."
+
+msgid ""
+"To add new contacts, join a room and create a contact add request by "
+"clicking on the star icon next to a user entry."
+msgstr ""
+"Pour ajouter des contacts, rejoignez une salle et faites une requête "
+"d'ajout de contact en cliquant sur l'étoile à côté de la fiche d'un utilisateur."
+
+msgid "Edit contact"
+msgstr "Modifier le contact"
+
+msgid "Edit"
+msgstr "Modifier"
+
+msgid "Name"
+msgstr "Nom"
+
+msgid "Remove"
+msgstr "Enlever"
+
+msgid "Refresh"
+msgstr "Actualiser"
+
+msgid "Save"
+msgstr "Enregistrer"
+
+msgid "Close"
+msgstr "Fermer"
+
+msgid "File sharing"
+msgstr "Partage de fichier"
+
+msgid "File is no longer available"
+msgstr "Ce fichier n'est plus disponible"
+
+msgid "Download"
+msgstr "Télécharger"
+
+msgid "Open"
+msgstr "Ouvrir"
+
+msgid "Unshare"
+msgstr "Ne plus partager"
+
+msgid "Retry"
+msgstr "Réessayer"
+
+msgid "Download failed."
+msgstr "Echec du téléchargement."
+
+msgid "Share a YouTube video"
+msgstr "Partager une vidéo Youtube"
+
+msgid "Share a file as presentation"
+msgstr "Partager une présentation"
+
+msgid "Share your screen"
+msgstr "Partager votre écran"
+
+msgid "Chat"
+msgstr "Chat"
+
+msgid "Contacts"
+msgstr "Contact"
+
+msgid "Mute microphone"
+msgstr "Couper le micro"
+
+msgid "Turn camera off"
+msgstr "Couper la caméra"
+
+msgid "Settings"
+msgstr "Paramètres"
+
+msgid "Loading presentation ..."
+msgstr "Chargement de la présentation..."
+
+msgid "Please upload a document"
+msgstr "Merci d'envoyer un document"
+
+msgid ""
+"Documents are shared with everyone in this call. The supported file types"
+" are PDF and OpenDocument files."
+msgstr ""
+"Les documents sot partagés avec tous les participants à cet appel."
+"Les document pdf et opendocument sont supportés"
```

Comment:
Missing leading whitespace (to have a blank after the `.` from the sentence before).